### PR TITLE
fix: use `:focus` pseudo class instead of `blocklyFocused`

### LINKED ISSUE #8326: Remove the `blocklyFocused` CSS class and use the `focused` CSS selector instead
1. Remove the `blocklyFocused` class from being assigned in [`menu.ts`](https://github.com/google/blockly/blob/f8025a1bdd12025ecb8830159509148ff27ad3c8/core/menu.ts#L160).
2. Remove the `blocklyFocused` class from being assigned in [`dropdowndiv.ts`](https://github.com/google/blockly/blob/f8025a1bdd12025ecb8830159509148ff27ad3c8/core/dropdowndiv.ts#L140).
3. Change the [styles in `css.ts`](https://github.com/google/blockly/blob/f8025a1bdd12025ecb8830159509148ff27ad3c8/core/css.ts#L122) to use the [`:focus` css pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus).

This change should be made against the `rc/v12.0.0` branch. This should be marked as a breaking change.

To work on this issue, comment below asking to be assigned.

### PR DESCRIPTION
**Pull request by Skye967**
… style to :focus

## The basics

- [x ] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change)

### Resolves

Fixes #8326

### Proposed Changes

Remove the blocklyFocused class from being assigned in menu.ts and dropdowndiv.ts
Change the styles in css.ts to use the :focus css pseudo class

Removed the blocklyFocused class from being assigned in menu.ts and dropdowndiv.ts
Changed the styles in css.ts to use the :focus css pseudo class

### Test Coverage

Not necessary 

### Documentation

No it won't affect anything. just some minor css changes

### CODE REVIEW

**Comment by Skye967** [comment]
Sorry about that, getting used to the naming conventions and I submitted the CLA. Other than that, do the changes look good?

**Comment by BeksOmega** [comment]
Hello! Thanks for the fix and for signing the CLA :D

It looks like your branch has a few extra commits. To remove the commits that aren't yours, you can rebase your branch on top of the latest `rc/v12.0.0` branch.  This will remove any commits that were added to `rc/v12.0.0` since you created your branch.

Here are the steps to do that:

1. **Fetch the latest changes from `rc/v12.0.0`:**

   ```bash
   git fetch upstream rc/v12.0.0
   ```

2. **Checkout your branch:**

   ```bash
   git checkout [your-branch-name]
   ```

3. **Start an interactive rebase:**

   ```bash
   git rebase -i upstream/rc/v12.0.0
   ```

4. **In the editor that opens, identify the commits that aren't yours.** These are likely the ones at the top of the list. 

5. **Change the word `pick` to `drop` for each commit you want to remove.** 

6. **Save and close the editor.**

8. **Force push your changes to your branch:** Note that force pushing is a dangerous operation because it overwrites commit history, so if you dropped the wrong commits you could lose work.

   ```bash
   git push -f origin [your-branch-name]
   ```

Once you've done this, the PR should only contain your commits.

Let me know if you have any questions or run into any issues!

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Marking this as changes requested.

File: core/css.ts [hunk 1]
```diff
@@ -119,7 +118,7 @@ let content = `
   box-shadow: 0 0 3px 1px rgba(0,0,0,.3);
 }
 
-.blocklyDropDownDiv.blocklyFocused {
```
Comment (BeksOmega) [inline core/css.ts:122]: There is another reference to `.blocklyFocused` in this file, could you remove it as well?

**Review by BeksOmega — CHANGES_REQUESTED** [review]
One comment, but changes to the dropdown div look good!

**Comment by Skye967** [comment]
Went of core css and made some changes.

**Review by BeksOmega — APPROVED** [review]
Sweet LGTM =) I'll get this merged once CI passes

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Can you fix the lint errors?

**Comment by Skye967** [comment]
I believe everything is good now.

**Review by BeksOmega — APPROVED** [review]

**Comment by BeksOmega** [comment]
Thanks for your hard work on this!

**Pull request merged**